src/Provider/TMSProvider.js

```diff
@@ -23,27 +24,34 @@ function preprocessDataLayer(layer) {
             max: 18,
         };
     }
+    layer.fx = layer.fx || 0.0;
 }
 
 function executeCommand(command) {
     const layer = command.layer;
     const tile = command.requester;
+    const isGlobeView = !command.view.wgs84TileLayer;
```

Comment:
Why do you need this?

Reply:
Forgot to remove it, sorry, it was my first fix for bad tiling when using a globe view.